lib/modules/manager/leiningen/extract.spec.ts

```diff
@@ -4,12 +4,16 @@ import { extractFromVectors, extractVariables, trimAtKey } from './extract';
 import { extractPackageFile } from '.';
 
 const leinProjectClj = Fixtures.get(`project.clj`);
+const leinParentProjectClj = Fixtures.get(`parent.clj`);
```

Comment:
move to test if only used once

Reply:
https://github.com/renovatebot/renovate/pull/29552/commits/ea17d69137415a3d853789be32753fb04b3e3565